shuffle-transfer/src/main/java/com/alibaba/flink/shuffle/transfer/ReadServerHandler.java

```diff
@@ -107,13 +107,13 @@ protected void channelRead0(ChannelHandlerContext ctx, TransferMessage msg) {
     @Override
     public void channelRegistered(ChannelHandlerContext ctx) throws Exception {
         super.channelRegistered(ctx);
-        NetworkMetrics.numReadingConnections().inc();
+        NetworkMetricsUtil.registerNumReadingConnections().inc();
```

Comment:
Why do we rename these metrics methods?
For example, here is counting for reading connections, is it better to use `numReadingConnections`?

Reply:
The rename is for consistency and easier understanding. We have two phases here, first register a metric, second update the metric. The register rename change is for easier understanding of first step. I agree that it is a little misleading here. I will split the two phases here into two line and make the Counter a global variable which will make it easier to understand.